tests/lib/tools/fs-state

```diff
@@ -1,38 +1,53 @@
 #!/bin/bash
 
+show_help() {
+	echo "usage: fs-state wait-for-file [filename] [iterations] [sleep-time]"
+	echo "usage: fs-state ensure-dir-exists [direname]"
```

Comment:
Could this be called `mock-dir` and be paired with `restore-dir`?

Reply:
done